ansible/roles_ocp_workloads/bookbag/tasks/pre_workload.yaml

```diff
@@ -0,0 +1,16 @@
+---
+# Implement your Pre Workload deployment tasks here
+- name: Check for required variables
+  assert:
+    that:
+    - guid is defined
```

Comment:
Didn’t you teach me the shorthand?
guid | default(“”)| length >0

Reply:
Yes, but I thought for showing an assertion error it would be useful for the user to see whether the variable was empty string or undefined as separate failed conditions.